res/css/views/rooms/_RoomPreviewCard.pcss

```diff
@@ -131,5 +131,9 @@ limitations under the License.
             padding: 14px 0;
             flex-grow: 1;
         }
+
+        .mx_AccessibleButton_kind_secondary {
+            border: 2px solid #0dbd8b;
+        }
```

Comment:
This should be accomplished by using the correct kind of `AccessibleButton`

Reply:
Thanks for the feedback! Can you clarify for me what is the correct kind `AccessibleButton` for use in this case?

Reply:
IIRC, `primary_outline` is the one, but you can find all of them in `AccessibleButton.tsx` :)

Reply:
@gefgu are you able to take a look at this?

Reply:
@t3chguy - made a commit using the `primary_outline` - I think it solves the issue now - thanks for pointing it out :)